### PR TITLE
Body scanner fix

### PR DESCRIPTION
**Pull request by DeadmonWonderland**
## Description of changes

Adds a proper tag in the scanner displays so they will now push to the body scan displays. Also gets rid of display that was causing some wierd fuckery.  Also removes a part of a table.

## Why and what will this PR improve

Fixes a bug. Fixes some wierd fuckery with displays, and slight housekeeping.

## Authorship

me

## Changelog

:cl: Deadmon
bugfix: Scans now push to the displays.
rscdel: Removes the display that was overlayed on another, resulting in some fuckery when you looked at it from the side.
/:cl:

### CODE REVIEW

**Comment by Mazianni** [comment]
fails CI because you use tag and not id_tag

**Comment by MarinaGryphon** [comment]
PR is nonfunctional, the wrong variable was used.

File: maps/torch/torch5_deck1.dmm [hunk 1]
```diff
@@ -1517,7 +1517,8 @@
 	pixel_y = 32
 	},
 /obj/machinery/body_scan_display{
-	pixel_y = 24
+	pixel_y = 24;
+	tag = "scanner_or"
```
Comment (MarinaGryphon) [inline maps/torch/torch5_deck1.dmm]:
```suggestion
	id_tag = "scanner_or"
```

File: maps/torch/torch5_deck1.dmm [hunk 2]
```diff
@@ -11726,7 +11721,8 @@
 	},
 /obj/machinery/optable,
 /obj/machinery/body_scan_display{
-	pixel_y = 24
+	pixel_y = 24;
+	tag = "scanner_or"
```
Comment (MarinaGryphon) [inline maps/torch/torch5_deck1.dmm]:
```suggestion
	id_tag = "scanner_or"
```

File: maps/torch/torch5_deck1.dmm [hunk 3]
```diff
@@ -12244,7 +12240,8 @@
 	pixel_y = 32
 	},
 /obj/machinery/body_scan_display{
-	pixel_y = 24
+	pixel_y = 24;
+	tag = "scanner_or"
```
Comment (MarinaGryphon) [inline maps/torch/torch5_deck1.dmm]:
```suggestion
	id_tag = "scanner_or"
```

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]
CI fixes.

**Review by MarinaGryphon — APPROVED** [review]

**Pull request merged**